mappings/document.js

```diff
@@ -147,10 +151,8 @@ var schema = {
       match_mapping_type: 'string',
       mapping: {
         type: 'string',
-        analyzer: 'peliasPhrase',
-        fielddata : {
-          loading: 'eager_global_ordinals'
-        }
```

Comment:
It appears in 2.4 fielddata is [enabled by default](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/fielddata.html#fielddata-format). How about we remove this change from this PR to keep it as small as possible, and let #302 handle it?